main/main.cpp

```diff
@@ -1929,6 +1930,8 @@ void Main::cleanup() {
 		memdelete(arvr_server);
 	}
 
+	ImageLoader::cleanup();
```

Comment:
I think this should probably work better in unregister_core_types() function

Reply:
It needs to occur before unregister_module_types which frees the image loaders. Moving it to unregister_core_types leaves dangling pointers in the ImageLoader array.